--- setup_kinetic.bash	2018-12-08 14:25:04.797495951 +0800
+++ setup_kinetic_new.bash	2018-12-15 20:44:38.340752941 +0800
@@ -76,17 +76,20 @@
 vcs import < ../../gazebo.repos
 
 cd ..
-catkin_make --pkg mav_msgs
-source devel/setup.bash
-catkin_make -j 1
-bash -c 'echo source `pwd`/devel/setup.bash >> ~/.bashrc'
+#catkin_make --pkg mav_msgs
+#source devel/setup.bash
+source $HOME/miniconda3/etc/profile.d/conda.sh
+conda activate openai-gym
+
+catkin_make --cmake-args -DCMAKE_CXX_STANDARD=11
+#bash -c 'echo source `pwd`/devel/setup.bash >> ~/.bashrc'
 echo "## ROS workspace compiled ##"
 
 # add own models path to gazebo models path
-if [ -z "$GAZEBO_MODEL_PATH" ]; then
-  bash -c 'echo "export GAZEBO_MODEL_PATH="`pwd`/../../assets/models >> ~/.bashrc'
-  exec bash #reload bashrc
-fi
+#if [ -z "$GAZEBO_MODEL_PATH" ]; then
+  #bash -c 'echo "export GAZEBO_MODEL_PATH="`pwd`/../../assets/models >> ~/.bashrc'
+  #exec bash #reload bashrc
+#fi
 
 # # Theano and Keras installation and requisites
 # cd ../
